src/js/components/modal.js

```diff
@@ -335,9 +311,9 @@ var Modal = (function() {
         if (activeModal.hasAttribute('data-modal-dialog')) return;
 
         close(activeModal.id);
-
-        activeTrigger.focus();
-
+        
+        if (activeTrigger !== null) activeTrigger.focus();
+        
```

Comment:
Remove unused indentation.

Reply:
Sorry, not sure which indentation you're talking about here. Everything is indented one tab stop (2 spaces) in from the `case KEYS.ecape:` switch statement above. How do you expect to see it indented instead?

Reply:
There are 8 spaces (`        `) on line 314 and 316, even though there is no code on those lines. I guess I leave that as a code style that you should manage, but it seems odd to have indentation on lines without code.

Reply:
Aaaah, ok, yeah, I see the spaces now. That reminds me that we do have a JS lint task set up that I should re-enable. I had removed when I was converting everything over to two spaces, and forgot to add it back. I'll do that as a separate PR and fix up any style errors it might be throwing. Probably would have caught those extra spaces.